lib/hsa/hsaTrackingBeamform.hpp

```diff
@@ -16,19 +16,21 @@
 #include <stdint.h> // for int32_t
 #include <string>   // for string
 /**
- * @class hsaBeamformPulsar
- * @brief hsaCommand to brute-force beamform for pulsar obs
+ * @class hsaTrackingBeamform
+ * @brief hsaCommand to brute-force beamform for tracking obs
  *
  *
- * This is an hsaCommand that launches the kernel (pulsar_beamformer) for
- * brute-force coherent beamforming and is most applicable to pulsar observations.
- * An array of phases (shape @c n_psr x @c n_elem x 2) is calculated by hsaPulsarUpdatePhase.cpp.
- * The default number of pulsar beams to be formed is 10. The phases are matrix
+ * This is an hsaCommand that launches the kernel (tracking_beamformer) for
+ * brute-force coherent beamforming and is most applicable to tracking observations.
+ * An array of phases (shape @c n_tracking x @c n_elem x 2) is calculated by
+ hsaTrackingUpdatePhase.cpp.
+ * The default number of tracking beams to be formed is 10. The phases are matrix
  * multiplied with the input data (shape @c n_samp x @c n_elem) and the output is of dimension
- * (@c n_samp x @c n_psr x @c n_pol x 2). Output data type is 4-4b int packed as char. Currently
+ * (@c n_samp x @c n_tracking x @c n_pol x 2). Output data type is 4-4b int packed as char.
+ Currently
```

Comment:
I'm not sure why, but the clang-format isn't adding an extra `* ` to the start of these lines.   Same on line `55` where it would be nice to also line up the new line with sentence above.